JSONSchema/genesis-filler-schema.json

```diff
@@ -0,0 +1,134 @@
+                            "description": "prestate account address with 0x prefix"
+                        }
+                    },
+                    "type": "object"
+                },
+                "genesis": {
+                    "additionalProperties": false,
+                    "properties": {
+                        "author": {
+                            "$ref": "#/definitions/AddressMaybePrefixOrEmpty"
+                        },
+                        "difficulty": {
+                            "$ref": "#/definitions/IntegerOrConfusedHex"
+                        },
+                        "extraData": {
+                            "$ref": "#/definitions/HexData"
+                        },
+                        "gasLimit": {
+                            "$ref": "#/definitions/IntegerOrConfusedHex"
+                        },
+                        "nonce": {
+                            "$ref": "#/definitions/IntegerOrConfusedHex"
+                        },
+                        "timestamp": {
+                            "$ref": "#/definitions/IntegerOrConfusedHex"
+                        }
+                    },
+                    "required": [
+                        "author",
+                        "difficulty",
+                        "extraData",
+                        "gasLimit",
+                        "nonce",
+                        "timestamp"
+                    ]
+                },
+                "params": {
+                    "additionalProperties": false,
+                    "properties": {
+                        "EIP150ForkBlock": {
+                            "$ref": "#/definitions/IntegerOrConfusedHex"
+                        },
+                        "EIP158ForkBlock": {
+                            "$ref": "#/definitions/IntegerOrConfusedHex"
+                        },
+                        "byzantiumForkBlock": {
+                            "$ref": "#/definitions/IntegerOrConfusedHex"
+                        },
+                        "chainID": {
+                            "$ref": "#/definitions/HexData"
+                        },
+                        "constantinopleForkBlock": {
+                            "$ref": "#/definitions/IntegerOrConfusedHex"
+                        },
+                        "homesteadForkBlock": {
+                            "$ref": "#/definitions/IntegerOrConfusedHex"
+                        },
+                        "miningMethod": {
+                            "$ref": "#/definitions/MiningMethod"
+                        }
+                    },
+                    "required": [
+                        "EIP150ForkBlock",
+                        "EIP158ForkBlock",
+                        "byzantiumForkBlock",
+                        "chainID",
+                        "constantinopleForkBlock",
+                        "homesteadForkBlock",
```

Comment:
Same for these -- if we make these required, we'll have to update the genesis spec to make future fork blocks also required -- otherwise it'll be inconsistent. So I would instead say that all forkblocks are implicitly `maxUint64` unless otherwise specified. Don't know how to express that in json spec though... 
A wild idea would be to have a separate section for those: `params/forks/xxx`.

Reply:
I guess I'll make these not required for now...